cmd/clusterctl/client/cluster/cert_manager_test.go

```diff
@@ -41,15 +41,27 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+const (
+	expectedHash    = "af8c08a8eb65d102ba98889a89f4ad1d3db5d302edb5b8f8f3e69bb992faa211"
+	expectedVersion = "v0.16.1"
```

Comment:
Sorry for asking another question on those lines, but I want to be sure this is not going against the goal of this PR.

Are those values some sort or fake values we use for tests or those values should be kept in sync we the current manifest?

If it is the first option, we are ok, but I would recommend to add a comment or make it explicit by renaming variables into "fake" something. if it is the second option, then might be we should explore options to fall back into the first case...

Reply:
you made a very good point, actually, I tried and unfortunately the answer is #2
so I will modify the test code (I think previous error are mostly hard code in test cases lead to error)
and give a try, thanks for point out

Reply:
the hash and version has been changed to `dummy` value 

and only one thing kept as we want to make sure we keep the comparison between version and current version , the test case is `Test_VersionMarkerUpToDate`

Reply:
thanks